### PR TITLE
Add sample file for Charge Links

### PR DESCRIPTION
**Pull request by prtandrup**
## Description

This PR adds a single sample file for Charge Links.
File is used in a domain test to verify its' validity.

When this PR gets merged, the 'EKSEMPELFILER TIL PRISER OG PRISLINKS' link on the public Energinet [page](https://energinet.dk/Energidata/DataHub/Dokumentation-og-regler) needs to be updated to: 
https://github.com/Energinet-DataHub/geh-charges/tree/main/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/TestFiles/Samples/

## References

* #0000

### CODE REVIEW

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/GreenEnergyHub.Charges.IntegrationTest.Core.csproj [hunk 1]
```diff
@@ -134,49 +134,52 @@ limitations under the License.
...
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="TestFiles\Charges\Samples\BundledChargeInformationSample.xml">
+    <Content Include="TestFiles\Charges\TariffBundleWithCreateAndUpdate.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="TestFiles\Charges\TariffBundleWithValidAndInvalid.xml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="TestFiles\Charges\Samples\BundledChargePriceSeriesSample.xml">
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/GreenEnergyHub.Charges.IntegrationTest.Core.csproj:140]: Nice, alphabetically sorted list of test-files - your "inspektor" colleague is a fan! The downside is that it is hard to understand, what actually changed in this pull request. Only one added xml?

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/TestFiles/Charges/ChargeDocument.cs [hunk 2]
```diff
@@ -33,14 +33,14 @@ public static class ChargeDocument
...
-        public const string ChargeInformationFeeMonthlySample = "TestFiles/Charges/Samples/ChargeInformationFeeMonthlySample.xml";
-        public const string ChargeInformationSubscriptionMonthlySample = "TestFiles/Charges/Samples/ChargeInformationSubscriptionMonthlySample.xml";
-        public const string BundledChargeInformationSample = "TestFiles/Charges/Samples/BundledChargeInformationSample.xml";
-        public const string ChargePriceSeriesSubscriptionMonthlySample = "TestFiles/Charges/Samples/ChargePriceSeriesSubscriptionMonthlySample.xml";
-        public const string ChargePriceSeriesFeeMonthlySample = "TestFiles/Charges/Samples/ChargePriceSeriesFeeMonthlySample.xml";
-        public const string ChargePriceSeriesTariffHourlySample = "TestFiles/Charges/Samples/ChargePriceSeriesTariffHourlySample.xml";
-        public const string BundledChargePriceSeriesSample = "TestFiles/Charges/Samples/BundledChargePriceSeriesSample.xml";
+        public const string ChargeInformationTariffHourlySample = "TestFiles/Samples/Charges/ChargeInformationTariffHourlySample.xml";
```
Comment (HenrikSommer) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/TestFiles/Charges/ChargeDocument.cs:36]: This means, the link on energinet.dk has to change - should it just link to the _testfiles/samples_ folder?

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/GreenEnergyHub.Charges.IntegrationTest.Core.csproj (hunk 1 above)
Comment (prtandrup) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/GreenEnergyHub.Charges.IntegrationTest.Core.csproj:140]: Can't take credit for the sorted list, I think it is the work of the IDE, but thanks anyway 😄 I have indeed only added one file and move the existing sample files around, but it looks weird here on GitHub 😄

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/TestFiles/Charges/ChargeDocument.cs (hunk 2 above)
Comment (prtandrup) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/TestFiles/Charges/ChargeDocument.cs:36]: Yep, once merged, I will request a link change on energinet.dk, so it becomes:

https://github.com/Energinet-DataHub/geh-charges/tree/main/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTest.Core/TestFiles/Samples/

**Review by HenrikSommer — APPROVED** [review]

**Pull request merged**